test/extensions/tracers/xray/tracer_test.cc

```diff
@@ -100,6 +100,14 @@ TEST_F(XRayTracerTest, NonSampledSpansNotSerialized) {
   span->finishSpan();
 }
 
+TEST_F(XRayTracerTest, BaggageNotImplemented) {
+  Tracer tracer{"" /*span name*/, std::move(broker_), server_.timeSource()};
+  auto span = tracer.createNonSampledSpan();
+  span->setBaggage("baggage_key", "baggage_value");
+  span->finishSpan();
+  ASSERT_EQ("", span->getBaggage("baggage_key"));
```

Comment:
nit: can you add a comment saying this isn't supported here as well to clarify testing for the empty value

Reply:
fixed in https://github.com/envoyproxy/envoy/pull/12260/commits/05dbaa50f5cc45f851bf881120f849f2d4e5e80f